pages/juan-ortizc/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body class="carousel">
+    <header class="header-carousel">
```

Comment:
I think it's better to create a section for the carrousel that is within the `body` instead of doing the `body` of the page as the absolute carrousel section.